src/equations/shallow_water_multilayer_1d.jl

```diff
@@ -54,10 +60,14 @@ struct ShallowWaterMultiLayerEquations1D{NVARS, NLAYERS, RealT <: Real} <:
        AbstractShallowWaterMultiLayerEquations{1, NVARS, NLAYERS}
     gravity::RealT   # gravitational constant
     H0::RealT        # constant "lake-at-rest" total water height
+    threshold_limiter::RealT    # threshold for the positivity-limiter
+    threshold_desingularization::RealT  # threshold for velocity desingularization
```

Comment:
From the default values below I see that here `threshold_limiter` now takes the default value of `threshold_wet` from the other shallow water equations structs. Plus, anywhere `threshold_wet` was used in `ShallowWaterEquationsWetDry1D` now `threshold_limiter` is used. Now the new threshold for Chertock's desingulartization procedure takes the place of the second threshold with a slightly larger default value compared to the old `threshold_limiter=500*eps()`. 

It is somewhat subpotimal that we have different thresholds across the equation structs. But the strategies are different so as long as everything is documented I suppose it is not the end of the world. If one runs the beach test case using the threshold defaults for `ShallowWaterEquationsWetDry1D` and then runs the same beach test case for the single layer version of `ShallowWaterMultiLayerEquations1D` is there any appreciable difference in the eyeball-norm?

Reply:
Yes this is confusing, as you wrote owed to the fact that we use different strategies to achieve positivity. However, since we use distinct names for the thresholds their use case should be clear and for most cases a user shouldn't have to interact with the thresholds at all, so I think it's not an issue.

Below is a comparison of the beach testcase using the standard elixir settings and default tolerances. The solution is almost identical (even though the SWE uses a HLL-flux, while the MLSWE uses a LLF-flux). The only visible difference is that the new strategy for the MLSWE (orange) doesn't suffer as much from velocity oscillations in the dry region as the SWE (blue).

Reply:
This is a good indication that the velocity desingularization is a "better" strategy because it avoids all the noise in the velocity in the dry region.